netbox/dcim/tests/test_models.py

```diff
@@ -439,6 +453,37 @@ def test_cable_cannot_terminate_to_an_existing_connection(self):
         with self.assertRaises(ValidationError):
             cable.clean()
 
+    def test_singlepos_rearport_connections(self):
```

Comment:
Needs a topology diagram like the other tests. Name should be `test_connection_via_single_position_rearport`.